domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/directionsquestionnaire/DirectionsQuestionnaire.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.hmcts.cmc.domain.models.directionsquestionnaire;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import java.util.List;
+import java.util.Optional;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import static java.util.Collections.emptyList;
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@Getter
+@EqualsAndHashCode
+public class DirectionsQuestionnaire {
+
+    private final RequireSupport requireSupport;
+
+    @NotNull
+    private final HearingLocation hearingLocation;
+    private final Witness witness;
+    @Size(min = 1)
+    private final List<ExpertReport> expertReports;
+    private final List<UnavailableDate> unavailableDates;
+    private final ExpertRequest expertRequest;
+
+    @Builder
```

Comment:
Better put this on the class since there's only one constructor

And other classes in this PR